src/Microsoft.ML.Transforms/Text/TextNormalizing.cs

```diff
@@ -215,6 +215,7 @@ public Mapper(TextNormalizingTransformer parent, DataViewSchema inputSchema)
 
             public bool CanSaveOnnx(OnnxContext ctx) => (_parent._keepDiacritics && _parent._keepNumbers && _parent._keepPunctuations);
 
+            //This Estimator requires OpSet 11
             public void SaveAsOnnx(OnnxContext ctx)
```

Comment:
The TextNormalizingEstimator has in its doc-comments a field for whether it is exportable to Onnx. There you can specify the required version number.